backend/ickly/settings.py

```diff
@@ -27,6 +27,19 @@
 
```

Comment:
you should set up a `secret_settings` file which is gitignored

then you can move the `SECRET_KEY` and database username/password to this file, and in the README just instruct the dev to create this file with personalized settings